config.json

```diff
@@ -245,6 +245,14 @@
       "plugin": "TeamRandomizer",
       "enabled": true,
       "command": "randomize"
+    },
```

Comment:
This needs to be in the template folders and generation commands ran,

Reply:
`squad-server/templates/config-template.json`?